src/types/dict.rs

```diff
@@ -134,35 +134,10 @@ impl PyDict {
 
     /// Gets an item from the dictionary.
     ///
-    /// Returns `None` if the item is not present, or if an error occurs.
+    /// Returns `Ok(None)` if the item is not present.
     ///
     /// To get a `KeyError` for non-existing keys, use `PyAny::get_item`.
```

Comment:
This reference to `get_item` does not make sense after the change, does it?

Reply:
It is still applicable because `PyAny::get_item` returns `PyResult<&PyAny>` and so nonexistent values need to raise.

I'll add an example to the doc to illustrate the difference.